integration/http/src/main/java/io/seata/integration/http/DefaultHttpExecutor.java

```diff
@@ -0,0 +1,157 @@
+
+/**
+ * The default http executer.
+ *
+ * @author wangxb
+ */
+public class DefaultHttpExecutor extends AbstractHttpExecutor {
+
+    private static DefaultHttpExecutor instance = new DefaultHttpExecutor();
+
+    private DefaultHttpExecutor() {
+    }
+
+    public static DefaultHttpExecutor getInstance() {
+        return instance;
+    }
+
+    @Override
+    public <T> void buildClientEntity(CloseableHttpClient httpClient, T paramObject) {
+
+    }
+
+    @Override
+    public <T> void buildGetHeaders(Map<String, String> headers, T paramObject) {
+
+    }
+
+
+    @Override
+    public <T> String initGetUrl(String host, String path, Map<String, String> querys) {
+
+        if (querys.isEmpty()) {
+            return host + path;
+        }
+        StringBuilder sbUrl = new StringBuilder();
+        sbUrl.append(host);
+        if (!StringUtils.isBlank(path)) {
+            sbUrl.append(path);
+        }
+
+        StringBuilder sbQuery = new StringBuilder();
+        Iterator queryKeys = querys.entrySet().iterator();
+
+        while (queryKeys.hasNext()) {
+            Map.Entry<String, String> query = (Map.Entry) queryKeys.next();
+            if (0 < sbQuery.length()) {
+                sbQuery.append("&");
+            }
+
+            if (StringUtils.isBlank(query.getKey()) && !StringUtils.isBlank(query.getValue())) {
+                sbQuery.append(query.getValue());
+            }
+
+            if (!StringUtils.isBlank(query.getKey())) {
+                sbQuery.append(query.getKey());
+                if (!StringUtils.isBlank(query.getValue())) {
+                    sbQuery.append("=");
+                    try {
+                        sbQuery.append(URLEncoder.encode(query.getValue(), "UTF-8"));
+                    } catch (UnsupportedEncodingException e) {
+                        throw new RuntimeException(e.getMessage());
+                    }
+                }
+            }
+        }
+
+        if (sbQuery.length() > 0) {
+            sbUrl.append("?").append(sbQuery);
+        }
+
+        return sbUrl.toString();
+
+    }
+
+    @Override
+    public <T> void buildPostHeaders(Map<String, String> headers, T t) {
+
+    }
+
+    @Override
+    public <T> StringEntity buildEntity(StringEntity entity, T t) {
+        return entity;
+    }
+
+    @Override
+    public <K> K convertResult(HttpResponse response, Class<K> clazz) {
+
+
+        if (clazz == HttpResponse.class) {
+            return (K) response;
+        }
+        return null;
+    }
+
+    public static void main(String[] args) {
+        String str = "{\n" +
+                "    \"name\":\"zhangsan\",\n" +
+                "    \"age\":15\n" +
+                "}";
+        Map<String, String> map = convertParamOfJsonString(str, Person.class);
+        System.out.println(map.toString());
+        Person person = JSON.parseObject(str, Person.class);
+        map = convertParamOfBean(person);
+        System.out.println(map.toString());
+
+
+    }
```

Comment:
Please clear the main method, if it is useful, use log instead of System.out.println, if you don't replace it, you will not pass the ci check.